azure-quantum/tests/unit/recordings/test_plugins_submit_qiskit_to_honeywell.yaml

```diff
@@ -48,7 +52,11 @@ interactions:
       Connection:
       - keep-alive
       User-Agent:
+<<<<<<< HEAD
```

Comment:
It looks like some merge conflicts made it into this branch.

Reply:
It is resolved now.